parlai/core/dialog_teacher.py

```diff
@@ -83,6 +83,10 @@ def __next__(self):
         if self.epochDone:
             raise StopIteration()
 
+    @property
+    def internal_states(self):
+        return super().internal_states + ['lastY']
```

Comment:
TIL about Python properties--cool! But it doesn't look like we're taking advantage of them at all, right? This is just removing `()` from after `a.internal_states`